src/ontology/cl-edit.owl

```diff
@@ -27343,11 +27339,12 @@ AnnotationAssertion(oboInOwl:created_by obo:CL_4023011 <http://orcid.org/0000-00
 AnnotationAssertion(oboInOwl:hasDbXref obo:CL_4023011 "xref (ILX:0770149)")
 AnnotationAssertion(rdfs:label obo:CL_4023011 "Lamp5 GABAergic cortical interneuron")
 EquivalentClasses(obo:CL_4023011 ObjectIntersectionOf(obo:CL_0000099 ObjectSomeValuesFrom(obo:RO_0002100 obo:UBERON_0000956) ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0061534) ObjectSomeValuesFrom(obo:RO_0002292 <https://identifiers.org/ensembl:ENSMUSG00000027270>)))
+
 # Class: obo:CL_4023015 (sncg GABAergic cortical interneuron)
 
 AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:33186530") obo:IAO_0000115 obo:CL_4023015 "A GABAergic neuron located in the cerebral cortex that expresses Gamma-synuclein")
-AnnotationAssertion(obo:IAO_0000115 obo:CL_4023015 "xref:(ILX:0770150)")
 AnnotationAssertion(oboInOwl:created_by obo:CL_4023015 <http://orcid.org/0000-0001-7258-9596>)
+AnnotationAssertion(oboInOwl:hasDbXref obo:CL_4023015 "xref:(ILX:0770150)")
```

Comment:
Should just be 

`AnnotationAssertion(oboInOwl:hasDbXref obo:CL_4023015 "ILX:0770150")`

Apologies if I've missed this elsewhere.   If so, please make consistent.